packages/ui/src/proposals/modals/AddNewProposal/components/TriggerAndDiscussionStep.tsx

```diff
@@ -24,7 +26,12 @@ export const TriggerAndDiscussionStep = () => {
   ])
 
   const addMemberToWhitelist = (member: Member) => {
-    setValue('triggerAndDiscussion.discussionWhitelist', [...discussionWhitelist, member], { shouldValidate: true })
+    if (count < 20){
+      setValue('triggerAndDiscussion.discussionWhitelist', [...discussionWhitelist, member], { shouldValidate: true })
+      setCount(count+1)
+    }else{
+      setSelected(true)
+    }
```

Comment:
Please extend existing validation on the select field like [here](https://github.com/Joystream/pioneer/blob/dev/packages/ui/src/proposals/modals/AddNewProposal/components/TriggerAndDiscussionStep.tsx#L75), see also [machine.ts](https://github.com/Joystream/pioneer/blob/dev/packages/ui/src/proposals/modals/AddNewProposal/machine.ts)